common/app/common/configuration.scala

```diff
@@ -350,6 +350,7 @@ class GuardianConfiguration extends Logging {
     lazy val d2Uid = configuration.getMandatoryStringProperty("discussion.d2Uid")
     lazy val frontendAssetsMap = configuration.getStringProperty("discussion.frontend.assetsMap")
     lazy val frontendAssetsMapRefreshInterval = 5.seconds
+    lazy val frontendAssetsVersion = "v1.1.0"
```

Comment:
it could, but because I'm expecting to change the assets version often, at least until everything is fully migrated, I don't want to create a new config every time.